entity-types/scheduledreport-scheduled_report/definition.yml

```diff
@@ -1,5 +1,5 @@
-domain: NR1
-type: SCHEDULED_REPORTS
+domain: SCHEDULEDREPORT
+type: SCHEDULED_REPORT
```

Comment:
This will end up expiring all the existing `NR1-SCHEDULED_REPORTS` entities. 

Is this domain/type being already used in production? Have we confirmed we can delete those existing entities?

Reply:
closing